### PR TITLE
Feature/gh 2222

### PR DESCRIPTION
**Pull request by mikelMHybris**
fixes GH-2222

### CODE REVIEW

File: projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-method.component.ts [hunk 1]
```diff
@@ -96,17 +96,30 @@ export class PaymentMethodComponent implements OnInit, OnDestroy {
           textDefaultPaymentMethod,
           textSelected,
         ]) => {
-          let ccImage: string;
+          let ccIcon: string;
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-method.component.ts]: Could you please move this logic to another function. So maybe like `private getCardIcon(code: string)` that returns the code and call it from here.

File: projects/storefrontstyles/scss/components/checkout/steps/payment/_payment-method.scss [hunk 2]
```diff
@@ -23,6 +23,15 @@
     }
   }
 
+  .cx-card-img-container {
+    .fab,
+    .fas {
+      &::before {
+        font-size: var(--cx-font-size, 66px);
```
Comment (LTiger14) [inline projects/storefrontstyles/scss/components/checkout/steps/payment/_payment-method.scss]: I feel like the icons were a bit smaller before. (like around 52px)

File: projects/storefrontlib/src/cms-components/misc/icon/icon.config.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
+export enum ICON_TYPES {
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/misc/icon/icon.config.ts]: Please delete this file

File: projects/storefrontstyles/scss/components/checkout/steps/payment/_payment-method.scss (hunk 2 above)
Comment (mikelMHybris) [reply projects/storefrontstyles/scss/components/checkout/steps/payment/_payment-method.scss]: the original size was 77x48. I'll dial it back to get the height to 48px.

File: projects/storefrontlib/src/lib/checkout/components/payment-method/payment-method.component.spec.ts [hunk 4]
```diff
@@ -27,6 +27,14 @@ import {
 import { CheckoutConfigService } from '../../checkout-config.service';
 import { defaultStorefrontRoutesConfig } from '../../../ui/pages/default-routing-config';
 
+@Component({
+  selector: 'cx-icon',
+  template: '',
+})
+export class MockCxIconComponent {
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/checkout/components/payment-method/payment-method.component.spec.ts]: You need to add this in the declarations array: (line 174-180)
```
declarations: [
        PaymentMethodComponent,
        MockPaymentFormComponent,
        MockCardComponent,
        MockSpinnerComponent,
        MockCxIconComponent,
      ],
```

File: projects/storefrontlib/src/shared/components/card/card.component.spec.ts [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
 import { CardComponent, Card, CardLinkAction } from './card.component';
 import { DebugElement } from '@angular/core';
 import { By } from '@angular/platform-browser';
 import { I18nTestingModule } from '@spartacus/core';
 
+@Component({
+  selector: 'cx-icon',
+  template: '',
+})
+export class MockCxIconComponent {
```
Comment (LTiger14) [inline projects/storefrontlib/src/shared/components/card/card.component.spec.ts]: You need to add this to the declarations array: (line 24)
```
declarations: [CardComponent, MockCxIconComponent],
```

**Pull request merged**